Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2476,6 +2476,13 @@ def to_context(self):
             }
             return ret_value
 
+        def to_readable(self):
+            dbot_score_to_text = {0: 'Unknown',
+                                  1: 'Good',
+                                  2: 'Suspicious',
+                                  3: 'Bad'}
+            return dbot_score_to_text[self.score]
+
```

Comment:
Please add tests. Make sure to test invalid cases as well.

Reply:
I added the happy path case.
Why should I add the invalid case? There could not be a dbot_score object with an invalid score (which is not 0, 1, 2, or 3).

Reply:
@ilaner Via the `__init__()` we don't expect it to be invalid, but a given code can change the `self.score` value. Please use `.get(, 'Undefined')` to ensure there won't be any errors in such a case.

Reply:
Fixed, and added a test for it